src/types/accountBillingStatus.js

```diff
@@ -7,13 +7,15 @@ class AccountBillingStatus {
     isTrial,
     trialPeriodEnds,
     isLegacy,
-    isGoodStanding = null
+    isGoodStanding = null,
+    isFree
```

Comment:
`isFree` should either come before the param that has a default value or should be given a valid default value itself. i'm not sure why `isGoodStanding` defaults to `null` but someone made that choice on purpose and it should be respected (ie we should be able to set `isFree` without needing to know the default value of `isGoodStanding`)

i'd be tempted to just default `isFree` to `null` as well. likely to distinguish between it being unset from being false.